menyer/for_goodness_shake/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+    background-color: wheat;
+    text-align: center;
+    font-family: 'Architects Daughter', cursive;
+    max-width: 720px;
+    margin: auto;
+}
+
+.container {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+}
+
+.menu {
+    z-index: -1;
+    color: white;
+    background: url("images/black-chalk-board-texture_89245-170.jpg");
+    background-size: cover;
+    border: 5px ridge grey;
+}
+
+.menuSection {
+    width: 33.33%;
+    height: 300px;
+    margin: 0;
+    padding: 0;
+}
+
+p {
+    font-size: 1.1em;
+    margin: 0;
+    padding: 0;
+}
+
+.menuItems {
+    position: absolute;
+    text-align: left;
+    left: 10px;
+}
+
+.hamburgare {
+    position: absolute;
+    left: 240px;
+}
+
+.tillbehor {
+    position: absolute;
+    left: 480px;
+}
+
```

Comment:
Avoid language mixing while coding. I recommend sticking with English always.

Reply:
This is fixed, don't know why I was using swedish for class names.